pkgs/by-name/sp/space-cadet-pinball/package.nix

```
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchzip,
+  cmake,
+  SDL2,
+  SDL2_mixer,
+  unrar-wrapper,
+}:
+
+let
+  assets =
+    (fetchzip {
+      url = "https://archive.org/download/SpaceCadet_Plus95/Space_Cadet.rar";
+      sha256 = "sha256-fC+zsR8BY6vXpUkVd6i1jF0IZZxVKVvNi6VWCKT+pA4=";
+      stripRoot = false;
+    }).overrideAttrs
+      (old: {
+        nativeBuildInputs = old.nativeBuildInputs ++ [ unrar-wrapper ];
+      });
```

```suggestion
      nativeBuildInputs = [ unrar ];
    };
```
https://nixos.org/manual/nixpkgs/unstable/#sec-pkgs-fetchers-fetchzip-examples